### DISCUSSION TITLE
How can I create a page editor with the fragments?

### DISCUSSION THREAD

**Discussion by wordbreak** [toeverything/blocksuite#6830] [Q&A]
First of all, thank you for this awesome project.

I want to use blocksuite to configure my editor,  and what I want is almost identical to the page editor in playground, but I don't want to use PageInfoFragment(a section that includes back links, tags, etc)

I've checked out the editor presets, but I don't think I can exclude what I want while using the presets.

How can I configure the blocksuite editor in the configuration I want (just title fragment and doc editor)?

Thanks

**Comment by raintoway** [comment]
localize the project as ur module, and accomplish what u want by update the code directly

**Reply by wordbreak** [reply]
Thanks. I know that's one of the workarounds, but is there any way I can use it without forking the project?

**Reply by raintoway** [reply]
i have not found the way to achieve what u want。and the ideology of blocksuite as shown in the pic below.
<img width="732" alt="截屏2024-04-27 13 24 55" src="https://github.com/toeverything/blocksuite/assets/38103330/879a9464-0da3-43ec-b721-e9fc136f90ec">
